client/api.go

```diff
@@ -170,6 +172,10 @@ type API struct {
 	Transfers *transfer.Client
 	// UsageRecords is the client used to invoke usage record related APIs.
 	UsageRecords *usagerecord.Client
+	// ValueLists is the client used to invoke /radar/value_lists APIs.
+	ValueLists *valuelist.Client
+	// ValueListItems is the client used to invoke /radar/value_list_items APIs.
+	ValueListItems *valuelistitem.Client
```

Comment:
I think we're generally prefixing these with the project namespace, so like `RadarValueLists`.